repos/system_upgrade/el7toel8/actors/bootentriesscanner/tests/test_bootentryscanner.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from leapp.libraries import stdlib
+from leapp.libraries.actor import bootentriesscanner
+from leapp.libraries.common.testutils import produce_mocked
+
+
+GRUBBY_INFO_ALL_STDOUT = '''
```

Comment:
Grubby output really starts with an empty line?

Reply:
Nice catch (although I should have noticed it myself - lesson learned), thank you.